docker-compose.yaml

```diff
@@ -7,19 +7,22 @@ services:
       - "9000:5000"
       - "9001:5001"
     volumes:
-      - thumbnails:/skyportal/static/thumbnails
+      - thumbnails:/skyportal/static/thumbnails:z
```

Comment:
Maybe add a content that :z is relevant for all shared content?

Reply:
done